portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/* Shows about text when the "About" button is pressed. */
+function showAbout () {
```

Comment:
This change includes changes from your other PR (https://github.com/shreyachatterjee00/step/pull/2/files).  Either this should be a clean branch that doesn't include the changes in the other PR (you'll have to merge when submitting), or should be diffbased off of the other PR (so I guess, merging into the other branch, shreyaDev, not master).

Reply:
So I merged the other PR into master, and then merged the new master into this branch so that the this branch would have the most recent changes. I think this should help? The diff files now look like they only include what was changed on this branch instead of the changes in the other PR